launch/launch/launcher.py

```diff
@@ -0,0 +1,154 @@
+import asyncio
+import signal
+from threading import Lock
+
+from launch import ProcessDescriptor
+from launch.output_handler import CompositeOutputHandler
+from launch.output_handler import ConsoleOutput
+from launch.exit_handler import DefaultExitHandler
+from launch.protocol import SubprocessProtocol
+
+
+class DefaultLauncher(object):
```

Comment:
The `(object)` isn't required in Python3 for "new" style classes. Leaving it doesn't hurt though.